functions/go/remove-local-config-resources/testdata/applied.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: demo-applied
+data:
+  foo: bar
```

Comment:
Please always add a newline at the end of a file.
Same for other files below.

Reply:
Acked